c2corg_api/search/sync.py

```diff
@@ -62,13 +64,27 @@ def remove_doc():
             # https://github.com/c2corg/v6_api/issues/89
             DBSession.refresh(document)
 
+        is_user_profile = isinstance(document, UserProfile)
+        if is_user_profile:
+            # set user login + full-name as document title so that it can
+            # be searched
+            user_login, user_name = DBSession. \
+                query(User.name, User.username). \
+                filter(User.id == document.document_id). \
+                first()
+            user_title = '{0} {1}'.format(user_name or '', user_login or '')
```

Comment:
c2corg is not just any community website! :P 
I know that that kind of issue was reported for the forum migration. Because some people use their real name as login and that it's not possible to change a login (unlike a forum pseudo or the guidebook name).
@gberaudo probably knows more about this, since he had a meeting with the client yesterday.